### DISCUSSION TITLE
Results screen fails to load when you get 1st place

### DISCUSSION THREAD

**Discussion by FamousPig** [ppy/osu#19335] [Q&A] [answered]
Opening as discussion because I am not 100% sure this is the exact issue, but it happened 2 out of 2 times so 🤷.

When I finished the beatmap and got first place, the score submitted but gameplay did not end and I never arrived at the results screen. Once I returned to song select I could see my score on the global leaderboard.

Map: https://osu.ppy.sh/beatmapsets/1506176#osu/3085224

logs: 
[runtime.log](https://github.com/ppy/osu/files/9174011/runtime.log) (it's the last map I played that run)
[network.log](https://github.com/ppy/osu/files/9174012/network.log)
[database.log](https://github.com/ppy/osu/files/9174013/database.log)
[updater.log](https://github.com/ppy/osu/files/9174014/updater.log)
[performance.log](https://github.com/ppy/osu/files/9174015/performance.log)

**Comment by frenzibyte** [comment]
Score position doesn't look to matter, as I've gotten first place multiple times without getting stuck at the ending.

The only thing I found interesting is that in both of your stuck plays, you look to have restarted a couple of times before achieving the score. Initial thoughts would be that something was disposed a bit late than usual, corrupting the state of the new gameplay. But I haven't got to reproduce this yet so can't say ([this is the only thing I found to be interesting](https://discord.com/channels/188630481301012481/188630652340404224/1000917128834195487)).

**Comment by peppy** [comment]
Did this only happen on that specific beatmap?

**Reply by frenzibyte** [reply]
According to the two logs, there were only two beatmaps OP encountered the issue on:
 1. https://osu.ppy.sh/beatmapsets/1242733#osu/2603005
 2. https://osu.ppy.sh/beatmapsets/1506176#osu/3085224

**Reply by peppy** [reply]
I'm still interested to know if the user can trigger it on an arbitrary beatmap or not. As far as I can see that isn't mentioned in the report.

**Reply by FamousPig** [reply]
Just tried reproducing this on several maps including the one from the OP, maps I had and didn't have previous scores on, maps with and without videos, with and without retrying, and I was unable to. 
So yeah not really sure what happened and why it happened 2 times in 1 session but it doesn't seem to happen anymore.

I haven't encountered this bug outside of those 2 times either.

**Reply by peppy** [reply]
Please post more details if you manage to reproduce this again.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/discussions/20455

**Reply by frenzibyte** [reply]
@FamousPig do you recall if this happened as a result of you pausing on the middle of the beatmap? and if so, do you recall having storyboard toggled on? and maybe not using 100% dim?

**Reply by FamousPig** [reply]
Sorry I don't remember if I paused or not. I am playing at 80% dim with video/sb enabled. The map I linked has a video but no storyboard, I'm not sure about the other case.
Haven't run into this issue since, so I don't really know more.

**Reply by frenzibyte** [reply]
Video requires a storyboard to be played. Either way thanks for the info, that confirms your issue to be caused by the thread linked above.
